src/core/server/saved_objects/import/utils.ts

```diff
@@ -85,10 +85,29 @@ export const updateDataSourceNameInVegaSpec = (
   return isJSONString
     ? JSON.stringify(parsedSpec)
     : stringify(parsedSpec, {
-        bracesSameLine: true,
-        keepWsc: true,
-        space: stringifiedSpacing,
-      });
+      bracesSameLine: true,
+      keepWsc: true,
+      space: stringifiedSpacing,
+    });
+};
```

Comment:
Can we address the linter error and remove unexpected changes?

Reply:
Would do